src/app/services/property-detail/property-detail.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from "@angular/core";
+import { HttpClient, HttpParams, HttpHeaders } from "@angular/common/http";
+import { Observable } from "rxjs";
+import { HttpService } from "../http.service";
+import { LocalStorageService } from '../local-storage.service'
+
+@Injectable({
+    providedIn: "root"
+})
+export class PropertyDetailService {
+    userToken = this.localStorageService.get('token', '');
+
+    constructor(
+        private http: HttpService,
+        private localStorageService: LocalStorageService) { 
+        
+        };
+
+    getProperty(slug): Observable<any> {
+
+        const httpOptions = {
+            headers: new HttpHeaders({
+            Authorization: `Bearer ${this.userToken}`
+
+            }),
+        };
+        const endpoint = `/properties/${slug}`
+    
+        return this.http.getRequestWithParams(endpoint, httpOptions);
+    }
+}
```

Comment:
Please Add a new line here

Reply:
@JamesMudidi the review app is okay now.